indexer/src/main/java/org/uniprot/store/indexer/subcell/SubcellularLocationStatisticsReader.java

```diff
@@ -17,6 +17,11 @@
 public class SubcellularLocationStatisticsReader
         implements RowMapper<SubcellularLocationStatisticsReader.SubcellularLocationCount> {
 
+    public static final String SUBCELLULAR_LOCATION_STATISTICS_QUERY =
```

Comment:
Can you please move this constant to SubcellularLocationStatisticsStep class, so you can make it private.